distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,105 @@
+* Admin Web User Interface
+** Web based install wizard.
+** Application grid to organize configurations.
+** Pluggable configuration pages for applications to simplify configurations for complex scenarios.
+** Metrics web application to view up to date system metrics.
+
+More information about {branding} is located at: https://tools.codice.org/wiki/display/{branding}[{branding} Home].
+
+=== Applications
+
+{branding} is comprised of several modular applications, to be installed or uninstalled as needed.
+
+{branding} Administrative Application::
+The administrative application enhances administrative capabilities when installing and managing {branding}. It contains various services and interfaces that allow administrators more control over their systems.
+{branding} Catalog Application::
+The {branding} Catalog provides a framework for storing, searching, processing, and transforming information.
+Clients typically perform query, create, read, update, and delete (QCRUD) operations against the Catalog.
+At the core of the Catalog functionality is the *Catalog Framework*, which routes all requests and responses through the system, invoking additional processing per the system configuration.
+{branding} Content Application::
+The {branding} Content application provides a framework for storing, reading, processing, transforming and cataloging data.
+{branding} Platform Application::
+The Platform application is considered to be a core application of the distribution. The Platform application has fundamental building blocks that the distribution needs to run. +
+These building blocks include subsets of: http://karaf.apache.org/[Karaf], http://cxf.apache.org/[CXF], http://karaf.apache.org/index/subprojects/cellar.html[Cellar], and http://camel.apache.org/[Camel]. +
+Included as part of the Platform application is also a Command Scheduler.
+The Command Scheduler allows users to schedule Command Line Shell Commands to run at certain specified intervals.
+
+{branding} Security Application::
+The Security application provides authentication, authorization, and auditing services for the {branding}. They comprise both a framework that developers and integrators can extend and a reference implementation that meets security requirements. More information about the security framework and how everything works as a single security solution can be found on the Managing Web Service Security page.
+{branding} Solr Catalog Application::
+The Solr Catalog Provider (SCP) is an implementation of the `CatalogProvider` interface using Apache Solr (http://lucene.apache.org/solr/) as a data store.
+{branding} Spatial Application::
+The {branding} Spatial Application provides KML transformer and a KML network link endpoint that allows a user to generate a View-based KML Query Results Network Link.
+{branding} Standard Search UI::
+The {branding} Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources.
```

Comment:
Shouldn't we mention federated queries in the _Catalog Application_ section as well?